### PR TITLE
Add celery to project

### PR DESCRIPTION
**Pull request by georgehelman**

### CODE REVIEW

File: config/settings/base.py [hunk 1]
```diff
@@ -260,6 +265,14 @@
 # http://docs.celeryproject.org/en/latest/userguide/configuration.html#task-soft-time-limit
 # TODO: set to whatever value is adequate in your circumstances
 CELERYD_TASK_SOFT_TIME_LIMIT = 60
+
+CELERY_BEAT_SCHEDULE = {
+    "clean_stale_data": {
+        "task": "dear_petition.petition.tasks.clean_stale_data.clean_stale_data_task",
+        "schedule": crontab(minute=0, hour="2"),
```
Comment (robert-w-gries) [inline config/settings/base.py:272]: Does this run every day at 2 AM? Can you add a comment to indicate when it's expected to run?

File: dear_petition/celery.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import os
+
+from celery import Celery
+
+# set the default Django settings module for the 'celery' program.
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "config.settings.local")
```
Comment (robert-w-gries) [inline dear_petition/celery.py:6]: Are there different settings for non-local environments we should be using?

**Review by robert-w-gries — COMMENTED** [review]
Couple minor questions but overall looks good to me. Colin might want to take a look as he likely has more Celery experience than me.

Liked the test addition btw!

File: config/settings/base.py (hunk 1 above)
Comment (georgehelman) [reply config/settings/base.py:274]: Sure. Yes, this schedules it to run at 2 AM EST each night.

File: dear_petition/celery.py (hunk 2 above)
Comment (georgehelman) [reply dear_petition/celery.py:6]: Yeah, it will be overwritten by Heroku config vars.

**Review by copelco — APPROVED** [review]

**Pull request merged**